treesapp_create.Rmd

```diff
@@ -6,47 +6,45 @@ In this tutorial we will demonstrate how to build a TreeSAPP reference package a
 
 The goals of this tutorial are:
 
-1) To learn how to use `treesapp create` to build a reference package for a simple protein family (XmoA) using the sequences acquired in the previous tutorial
-2) Learn how to assess whether the reference package contains only homologous sequences using `treesapp purity`
-3) Learn how to view a reference package's phylogenetic tree using the interactive Tree of Life (iTOL)
+1) To learn how to use `treesapp create` to build a reference package for a simple protein family (XmoA) using the sequences acquired in the previous tutorial.
+2) Learn how to assess whether the reference package contains only homologous sequences using `treesapp purity`.
+3) Learn how to view a reference package's phylogenetic tree using the interactive Tree of Life (iTOL).
 
 ## Making the reference package
 
 
-In the [Acquiring golden data] tutorial, PmoA & AmoA sequences were downloaded from [TIGRFAM](http://tigrfams.jcvi.org/cgi-bin/index.cgi), [EggNOG](http://eggnog5.embl.de/#/app/home), and [FunGene](http://fungene.cme.msu.edu/). For now we'll just use the TIGRFAM seed sequences and EggNOG sequences as these are best curated. Let's call this our seed reference package. The steps involved are:
+In the [Acquiring reference sequence data] tutorial, PmoA & AmoA sequences were downloaded from [EggNOG](http://eggnog5.embl.de/#/app/home) and [UniProt](https://www.uniprot.org/). Here, we will use the EggNOG sequences.
+ <!-- as well as seed sequences from [TIGRFAM](http://tigrfams.jcvi.org/cgi-bin/index.cgi) --- a database that is no longer updated but was well curated -->
```

Comment:
This is actually needed, at least the 2 TIGRFAM files are used below when testing purity

Reply:
The TIGRFAM files used for `treesapp purity` are different than those originally used for `treesapp create`. The files used to test purity are a fasta file containing seed sequences for all families in TIGRFAM whereas those originally used to create the reference package only contained sequences for pmoA and amoA.